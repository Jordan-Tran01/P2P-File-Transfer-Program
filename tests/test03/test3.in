PEERS
CONNECT 127.0.0.1:9856
PEERS