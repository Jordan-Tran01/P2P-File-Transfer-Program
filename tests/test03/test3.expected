Connection established with peer
Already connected to peer
Connected to:

1. 127.0.0.1:9856
